app/views/alchemy/admin/pages/_new_page_form.html.erb

```diff
@@ -4,8 +4,9 @@
   <%= f.input :page_layout,
     collection: @page_layouts,
     label: Alchemy.t(:page_type),
-    include_blank: Alchemy.t('Please choose'),
+    include_blank: @page_layouts.length == 1 ? nil : Alchemy.t('Please choose'),
     required: true,
+    selected: @page_layouts.length == 1 ? @page_layouts.first : nil,
```

Comment:
`page_layouts.first.presence` instead would do the trick

Reply:
nice catch will fix next round

Reply:
Actually I don't think i can change this as i select first only if length == 1 - Not if first is present